frontend/public/components/modals/configure-update-strategy-modal.tsx

```diff
@@ -20,12 +23,14 @@ export const getNumberOrPercent = (value) => {
 };
 
 export const ConfigureUpdateStrategy: React.FC<ConfigureUpdateStrategyProps> = (props) => {
+  const { showDescription = true } = props;
   return (
     <>
-      <div className="co-m-form-row">
-        <p>How should the pods be replaced when a new revision is created?</p>
-      </div>
-
+      {showDescription && (
+        <div className="co-m-form-row">
+          <p>{UPDATE_STRATEGY_DESCRIPTION}</p>
+        </div>
+      )}
```

Comment:
Needed to override default description when one is provided through a descriptor or schema property.